Modules/_operator.c

```diff
@@ -889,7 +889,7 @@ _operator__compare_digest_impl(PyObject *module, PyObject *a, PyObject *b)
 /* helper function to make typing.NewType.__call__ method faster */
 
 static PyObject *
-functools_idfunc(PyObject* self, PyObject* x) {
+operator_idfunc(PyObject* self, PyObject* x) {
```

Comment:
```suggestion
_operator__idfunc(PyObject* self, PyObject* x) {
```
(Following the "convention" in file, e.g. `_operator__compare_digest_impl`.)